diff --git a/pipelines_ootd/unet_garm_2d_blocks.py b/pipelines_ootd/unet_garm_2d_blocks.py
index 06dddd9..974d989 100644
--- a/pipelines_ootd/unet_garm_2d_blocks.py
+++ b/pipelines_ootd/unet_garm_2d_blocks.py
@@ -26,7 +26,7 @@ from diffusers.utils import is_torch_version, logging
 from diffusers.utils.torch_utils import apply_freeu
 from diffusers.models.activations import get_activation
 from diffusers.models.attention_processor import Attention, AttnAddedKVProcessor, AttnAddedKVProcessor2_0
-from diffusers.models.dual_transformer_2d import DualTransformer2DModel
+from diffusers.models.transformers.dual_transformer_2d import DualTransformer2DModel
 from diffusers.models.normalization import AdaGroupNorm
 from diffusers.models.resnet import Downsample2D, FirDownsample2D, FirUpsample2D, KDownsample2D, KUpsample2D, ResnetBlock2D, Upsample2D
 # from diffusers.models.transformer_2d import Transformer2DModel
diff --git a/pipelines_ootd/unet_vton_2d_blocks.py b/pipelines_ootd/unet_vton_2d_blocks.py
index 57e2555..0fc7425 100644
--- a/pipelines_ootd/unet_vton_2d_blocks.py
+++ b/pipelines_ootd/unet_vton_2d_blocks.py
@@ -26,7 +26,7 @@ from diffusers.utils import is_torch_version, logging
 from diffusers.utils.torch_utils import apply_freeu
 from diffusers.models.activations import get_activation
 from diffusers.models.attention_processor import Attention, AttnAddedKVProcessor, AttnAddedKVProcessor2_0
-from diffusers.models.dual_transformer_2d import DualTransformer2DModel
+from diffusers.models.transformers.dual_transformer_2d import DualTransformer2DModel
 from diffusers.models.normalization import AdaGroupNorm
 from diffusers.models.resnet import Downsample2D, FirDownsample2D, FirUpsample2D, KDownsample2D, KUpsample2D, ResnetBlock2D, Upsample2D
 # from diffusers.models.transformer_2d import Transformer2DModel
